lib/python/residuals.py

```diff
@@ -61,7 +61,7 @@ def read_residuals(filename="resid2.tmp"):
         not (reclen==struct.calcsize(rectype))):
         print("Warning:  possibly reading residuals incorrectly... don't understand record size")
     infile.seek(0, 0) # position at file start
-    r.numTOAs = filelen / reclen
+    r.numTOAs = int(filelen / reclen)
```

Comment:
@ryanslynch Shouldn't that be filelen // reclen since filelen and reclen should be ints?
```suggestion
    r.numTOAs = filelen // reclen
```

Reply:
Yes, that is the better way